js/src/stats.js

```diff
@@ -2,8 +2,13 @@ import { default as jStat } from 'jstat'
 import { default as stdlib } from '@stdlib/stdlib'
 
 // Aliases to reduce typing and allow for better minification
-const slspecial = stdlib.math.base.special
-const slstats = stdlib.math.statistics || stdlib.math.stats
+// ATTENTION: this introduces a dependency to stdlib during loading time
+// This can cause problems with cyclic dependencies, as well
+// as with loading order
+// I.e. stdlib must be loaded before mini-core, and stdlib must not
+// depend on mini-core
+// TODO: we should discuss if we want to avoid this in function libraries
+// const slstats = stdlib.math.statistics || stdlib.math.stats
```

Comment:
This is a workaround for what I think is a bug in the stdlib exports (which differs between cjs and umd bundles). This workaround was necessary to get tests to run in both node and browser.